application/api/.eslintrc.json

```
@@ -0,0 +1,16 @@
+{
+  "rules": {
+    "strict": [
+      "error",
+      "never"
+    ]
+  },
+  "globals": {
+    "BigInt": true,
+    "application": true,
+    "context": true,
+    "api": true,
+    "gs": false,
+    "Duration": false
```

```suggestion
    "BigInt": "readable",
    "application": "writable",
    "context": "writable",
    "api": "writable",
    "gs": "readable",
    "Duration": "readable"
```

To check for accidental overrides and make clearer syntax.
(https://eslint.org/docs/user-guide/configuring#specifying-globals)

---

I'll change all keys to "readable"